src/auth-service/utils/email.msgs.js

```diff
@@ -48,8 +48,8 @@ module.exports = {
       `Your temporary password is: ${password}\n\n` +
       `You can always change your password in your account settings after login\n` +
       `Follow this link to access the dashboard right now: ${constants.LOGIN_PAGE}\n` +
-      "A guide to using the platform will be found under the Documentation section of the platform\n" +
-      `Demos for using our main features can be found on our Youtube channel here: ${constants.YOUTUBE_CHANNEL}\n\n\n\n` +
+      "A guide to using the platform will be found under the Documentation section of the platform\n\n\n\n" +
```

Comment:
just removing the parts of the Youtube link since we are not actively managing the content on that channel.